packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -801,18 +815,26 @@ function parseTrimmedStack(rootStack: any, hook: HookLogEntry) {
     // Something went wrong. Give up.
     return null;
   }
-  return hookStack.slice(primitiveIndex, rootIndex - 1);
+  return [
+    hookStack[primitiveIndex - 1],
+    hookStack.slice(primitiveIndex, rootIndex - 1),
+  ];
```

Comment:
Where did this come from? Looking at the code, you are trying to use the `functionName` from the primitive frame, but we didn't have this before, is it a regression for some cases?